modules/040-node-manager/hooks/internal/v1/nodegroup.go

```diff
@@ -67,6 +67,9 @@ type NodeGroupSpec struct {
 	// Type of nodes in group: CloudEphemeral, CloudPermanent, CloudStatic, Static. Field is required
 	NodeType NodeType `json:"nodeType,omitempty"`
 
+	// Maximum drain time of CloudEphemeral machines in seconds
+	NodeDrainTimeout *float64 `json:"nodeDrainTimeout,omitempty"`
```

Comment:
Why float64, wen in openapi spec we have integer? With limiting i think we can use int here

Reply:
I had some difficulty with unmarshaling, will fix this